packages/askar-react-native/package.json

```diff
@@ -41,12 +41,12 @@
   },
   "devDependencies": {
     "react": "18.2.0",
-    "react-native": "0.74.1",
+    "react-native": "*",
```

Comment:
shouldn't this reflect the tested/dev version of react native? I think it is currently resolving to `0.76.6`.  I don't think this is a blocker. but I think `*` might get inadvertently updated to an unexpected major/minor version.

Reply:
I updated it to the same as the peerdep, `>= 0.71`. I prefer to be optimistic here and to say that 0.77 and 0.78, etc. will be supported from a version perspective. Otherwise you'll get a bit too annoying messages from `npm` that warn you that it might not work.